examples/module/imu/readme.md

```diff
@@ -90,3 +90,5 @@ Note: change `DATAROOT` to the folder you select.
         --datadrive DATADRIVE [DATADRIVE ...]
                                 data sequences
         --load_ckpt
+
+![imu-correction-result](./IMUcorrector.png)
```

Comment:
use links instead of putting images in code space.

refer to [this](https://github.com/pypose/pypose/blob/8c59ab67f208b9b3183f4ebf7afe2885f055e632/README.md?plain=1#L3). (directly copy and past images into github code editor will work)